monkey/common/event_serializers/event_serializer_registry.py

```diff
@@ -0,0 +1,17 @@
+from common.event_serializers import IEventSerializer
+
+
+class EventSerializerRegistry:
+    """
+    Registry for event serializers
```

Comment:
Once the methods are modified to accept classes, an example or further explanation here would be good.

Reply:
Still needed.